### PR TITLE
Dev 2045

### PR DESCRIPTION
**Pull request by superdev0714**
Added to use local/session storage for decrypted Subject
1. Workflow
- First Login (or the current users without this feature)
- Asking to use this feature or not
- If select Yes and login is `remember_me`, will use local storage for saving the decrypted subject.
- If select Yes and login is not `remember_me`, will use sessionStorage for saving the decrypted subject,
  This data would be removed by closing the broswer.
- if select No, nothing happened.

2. Needs to update 
 (Please confirm and let me know your thought @The-Hidden-Hand )
- Needs to update the dialog description to ask whether to use this feature or not (Signup, First Login)

**Current Title:**

`Using cache for decrypted subject`
**Current Description:**

`We provide the feature to store the decrypted subject on local or session storage for improve the speed. If you trust this machine, you can use this feature.`
image

- Needs to update the setting title on Settings/Security/Encryption

image

- Needs to update `Learn More` page on wordpress app.

Current referring URL: `https://ctemplar.com/help/answer/what-to-use-local-cache/`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "map.prototype.tojson": "^0.1.3",
     "moment": "^2.22.2",
     "moment-timezone": "^0.5.27",
+    "ngrx-store-localstorage": "^10.1.1",
```
Comment (The-Hidden-Hand) [inline package.json:57]: Do we need this dependency? @superdev0714

**Comment by The-Hidden-Hand** [comment]
I think a modal is too disruptive at this point, I was saying a checkbox or switch under remember me could do it.

What do you think? @superdev0714 @Godfry

**Comment by Godfry** [comment]
I think this is a great idea, we just need some finalized wording for this right @The-Hidden-Hand ?
Is this possible for the content of emails also? @superdev0714

**Comment by superdev0714** [comment]
> I think a modal could be disruptive at this point, not sure, I was saying that a checkbox or switch under "Remember me" could do it.
> 
> What do you think? @superdev0714 @Godfry

Yes, placing on the login page is also good idea.
Due to the condition that the users can't understand what this feature does exactly, we need to add the semi detailed explanation of this feature on the login page as well.
What is your idea for it, I don't think adding the switch with only title like `Using Cache for decrypted subject` is good.

**Comment by superdev0714** [comment]
> I think this is a great idea, we just need some finalized wording for this right @The-Hidden-Hand ?
> Is this possible for the content of emails also? @superdev0714

Yes, we can apply this on the decrypted content.

**Pull request merged**